docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -40,6 +49,20 @@ You can also analyze events from <<timelines-ui,Timelines>>
 
 Inside the visual analyzer, each cube represents a process (i.e. an executable file or network event). Click and drag in timeline view to see all process relationships. 
 
+To help understand what field were used to create the process, select the **Process Tree** to view the schema that created the graphical view. The fields included are:
+
+-`SOURCE`: Can be either `endpoint` or `winlogbeat`.
+- `ID`: What KQL entity created the process.
+- `EDGE`: The parent of the process ID.
```

Comment:
The schema is really meant to describe which fields are being used from the documents to represent the nodes and edges in the graph.

Maybe something like: **`EDGE`: The field from the events used to indicate the parent and child relationship between nodes**

Reply:
Agree with changing the description of edge, maybe even instead of "indicate the parent and child relationship" "indicate the  association between two nodes" or something even more generic.